src/css/flags/touch.less

```diff
@@ -21,5 +21,8 @@
     .jw-display-icon-container {
         pointer-events: none;
     }
-}
 
+    &.jwplayer&.jw-state-paused .jw-display-icon-container {
```

Comment:
Another little trick that will help us in the long run is to keep track of "why" things are so specific. For example see this -> https://github.com/jwplayer/jwplayer/blob/master/src/css/flags/audioplayer.less#L33